kubernetes/helm_charts/core/nginx-public-ingress/values.j2

```diff
@@ -190,15 +190,15 @@ proxyconfig: |-
       # Enabling caching
       # caching include Accept-Encoding header also, to provide gziped or plain content as per request
       proxy_cache_key "$http_accept_encoding|$request_uri|$request_body";
-      proxy_cache framework_cache;
+      proxy_cache {{proxy_cache_path.medium_cache.keys_zone.split(':') | first}};
```

Comment:
If in future, we decide to change this location block from medium cache to small cache, this code needs to be updated. We don't have an option to override this currently.